### PR TITLE
creation the  module adoption form question CRUD

### PR DESCRIPTION
**Pull request by RosneySoto**

### CODE REVIEW

File: packages/server/src/modules/adoption-form/controller.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      };
+   } catch (e) {
+      console.log(e);
+      return {
+         status: 500,
+         message: "Unexpected controller error",
+         detail: e,
+      };
+   }
+}
```
Comment (iocevelasco) [inline packages/server/src/modules/adoption-form/controller.ts:83]: add line at the bottom

File: packages/server/src/modules/adoption-form/network.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+         };
+      })
+      .catch((e) => {
+         console.log(e);
+         res.status(500).send("Unexpected Error");
+      });
+});
+
+
+export default router;
```
Comment (iocevelasco) [inline packages/server/src/modules/adoption-form/network.ts:96]: add line at the bottom

**Comment by RosneySoto** [comment]
line is already added at the bottom

**Comment by RosneySoto** [comment]
the form module documentation is added

**Pull request merged**